modules/utxo-lib/src/bitgo/signature.ts

```diff
@@ -360,3 +370,41 @@ export function verifySignature(
 
   return signatureVerifications.length > 0 && signatureVerifications.every((v) => v.signedBy !== undefined);
 }
+
+export function signInput2Of3(
+  txBuilder: UtxoTransactionBuilder,
+  vin: number,
+  scriptType: ScriptType2Of3 | 'p2shP2pk',
+  pubkeys: Triple<Buffer>,
+  keyPair: bip32.BIP32Interface,
+  cosigner: Buffer,
+  amount: number
+): void {
+  let prevOutScriptType;
+  let redeemScript;
+  let witnessScript;
+  let witnessValue: number | undefined = amount;
+
+  if (isScriptType2Of3(scriptType)) {
+    prevOutScriptType = scriptType2Of3AsPrevOutType(scriptType);
+    ({ redeemScript, witnessScript } = createOutputScript2of3(pubkeys, scriptType));
+  } else if (scriptType === 'p2shP2pk') {
+    prevOutScriptType = 'p2sh-p2pk';
+    ({ redeemScript, witnessScript } = createOutputScriptP2shP2pk(keyPair.publicKey));
+    witnessValue = undefined;
```

Comment:
the support for `p2shP2pk` makes the name of the function slightly inaccurate

maybe I should rename it to just `signInput`